### PR TITLE
Game UI updates; initial campaign tracking

### PR DESCRIPTION
**Pull request by ar0ch**
https://weitz-covid-dev.appliedbinf.com/?utm_source=dev

### CODE REVIEW

File: COVID19-Event-Risk-Planner/server/risk-game-reactivity.R [hunk 1]
```diff
@@ -86,39 +91,59 @@ observeEvent(input$setGeo, {
...
+      input = "checkbox",
       inputPlaceholder = "I am over 18 and in the US",
-      title = "Are you over 18 and in the US?",  text = paste0(
-        "Users under 18 and/or who reside outside the US",
+      title = "Are you over 18 and in the US?",
+      text = paste0(
+        "Users under 18 and/or those who reside outside the US",
         " are encouraged to use the risk prediction tools, ",
-        "but unfortunately we cannot save your survey feedback.  ",
+        "unfortunately we cannot save your survey feedback.  ",
```
Comment (ar0ch) [inline COVID19-Event-Risk-Planner/server/risk-game-reactivity.R]: Add the "but" back in

File: COVID19-Event-Risk-Planner/server/risk-game-reactivity.R [hunk 2]
```diff
@@ -1,76 +1,81 @@
...
+      inputId = "game_consent",
       inputPlaceholder = "I am over 18 and in the US",
-      title = "Are you over 18 and in the US?", text = paste0(
-        "Users under 18 and/or who reside outside the US",
+      title = "Are you over 18 and in the US?",
+      text = paste0(
+        "Users under 18 and/or those who reside outside the US",
         " are encouraged to use the risk prediction tools, ",
-        "but unfortunately we cannot save your survey feedback.  ",
+        "unfortunately we cannot save your survey feedback.  ",
```
Comment (ar0ch) [inline COVID19-Event-Risk-Planner/server/risk-game-reactivity.R]: Add "but" back in

File: COVID19-Event-Risk-Planner/server/risk-game-reactivity.R [hunk 3]
```diff
@@ -1,76 +1,81 @@
...
 }, ignoreNULL = T)
 
 output$location_selector <- renderUI({
   if (input$geolocation) {
     HTML("<p class='loc-text success'>Detected your location automatically</p>")
-    # input$geolocation <- FALSE
   }
   else{
-    HTML("<p class='loc-text'><b>Please choose your location below</b></p>")
+    HTML("<p class='loc-text'>Please choose your location below</p>")
```
Comment (ar0ch) [inline COVID19-Event-Risk-Planner/server/risk-game-reactivity.R:66]: I prefer this unbolded, double check with @sjbeckett

File: COVID19-Event-Risk-Planner/ui/risk-game-tab.R [hunk 4]
```diff
@@ -41,23 +41,37 @@ risk_game_tab = tabPanel(
...
-          label = "Select state"
-        ),
-        selectizeInput("risk_county", choices = NULL, label = "Select county"),
-        uiOutput("guessdf")
-        # )
+        div(style = "height: 25px;"),
+        div(
+          class = "well fake-sidebar",
+          HTML(
+            "<p class='intro-text'><a href='https://duke.qualtrics.com/jfe/form/SV_0SZR4fPxyUAg9Ke', rel='noopener' target='_blank'>Fill out this 5-minute survey</a> for a chance to win a $100 Amazon girf card!</p>"
```
Comment (ar0ch) [inline COVID19-Event-Risk-Planner/ui/risk-game-tab.R]: Is this the latest quiz link @sjbeckett?  Its the newest one in my emails

File: COVID19-Event-Risk-Planner/server/risk-game-reactivity.R (hunk 3 above)
Comment (sjbeckett) [reply COVID19-Event-Risk-Planner/server/risk-game-reactivity.R:66]: I think users may actually see this section last on the page (due to sizing of other elements), yet we would like for them to interact with it first. 

I think making this text larger would help - probably as large at the quiz question text. Bold/color may also help it stand out. What do you think?

**Comment by ar0ch** [comment]
Review requested: https://3.basecamp.com/4956471/buckets/21333742/messages/4349415119

**Comment by ar0ch** [comment]
Closed in favor of #52

**Pull request closed without merging**